src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckContent.java

```diff
@@ -110,11 +109,16 @@
 
     public static final String LABEL_BOM_COMPONENT = "Bom Component";
 
-    // TODO remove this
+    public static final String LABEL_REMEDIATION_FIX_PREVIOUS = "Remediation Fixes Previous Vulnerabilies";
+    public static final String LABEL_REMEDIATION_CLEAN = "Remediation Without Vulnerabilies              ";
+    public static final String LABEL_REMEDIATION_LATEST = "Remediation Latest Version                    ";
```

Comment:
Strange spacing here and I'm not sure why.

Reply:
The spacing is an attempt to get the labels to line up so the values all display in a straight line.